### ISSUE TITLE
More frequent screen updates in ambient mode

### ISSUE THREAD

**Issue by moneytoo** [fodawim/Facer-Support#104]
How about more frequent updates in ambient mode? 

Why? In my case for catching bus(es) every day, I need better than minute resolution without the need to twist my wrist. For me, 1 minute makes a difference. I made native proof of concept watch face ([Everyday Watch Face](https://play.google.com/store/apps/details?id=com.smartmadsoft.wear.face.everyday)) which uses AlarmManager to schedule wakeups every 20 seconds. I still return from work in the evening with 60-70% of battery left on my LG G Watch so there's not really any much additional battery drain.

It doen't seem that many watch faces care about seconds but maybe someone out there feels the same?

**Comment by mbecroft** [comment]
Agreed 100%. I'd love my watch to update every 10 seconds, say, with as you point out minimal battery impact.
